Validating the mobile number format of the user (#2342)
wso2/docs-is: en/docs/learn/configuring-sms-otp.md

    SendOTPDirectlyToMobile = false
    redirectToMultiOptionPageOnFailure = false
    ```

## Validating the mobile number format of the user

Follow the steps given below to validate the mobile number of the users in
WSO2 IS.    

!!! note

> 	**CaptureAndUpdateMobileNumber** should be enabled in SMS OTP Configuration Properties.

Reviewer: ```suggestion
	The `CaptureAndUpdateMobileNumber` property in the SMS OTP configuration properties should be set to **true**. 
```

> 1.  Go to the **SMS OTP Configuration** under **Federated

Reviewer: ```suggestion
1.  Expand **SMS OTP Configuration** under **Federated
```

	Authenticators**.

> 2.  Enter the regex pattern you want to validate when user enters the mobile number on 
> 	mobile number registration page.

Reviewer: ```suggestion
2.  2.  Enter the regex pattern you wish to validate the mobile number against when the user enters the mobile number on the registration page.
```

	![add-mobile-number-regex-pattern](../assets/img/tutorials/sms-otp/add-mobile-number-regex-pattern.png)

	!!! info

> 		-   If the entered mobile number to be validated against E.164 format, enter regex pattern as **\\+[1-9]\\d{1,14}\$**

Reviewer: ```suggestion
		 For example, if the entered mobile number is to be validated against the **E.164** format, enter the regex pattern as `\\+[1-9]\\d{1,14}\$`. 
```

> 3.  Enter the error message you want to show to user when user enters the mobile number in incorrect format.

Reviewer: ```suggestion
3.  Enter the error message that should be displayed to the user when the mobile number is entered in the incorrect format.
```
